packages/admin/admin/src/common/buttons/save/SaveButton.tsx

```diff
@@ -1,14 +1,74 @@
 import { Check, Error, Error as ErrorIcon, Save, ThreeDotSaving } from "@comet/admin-icons";
-import { Button, ButtonClassKey, ButtonProps, ComponentsOverrides, Theme } from "@mui/material";
-import { WithStyles, withStyles } from "@mui/styles";
-import { ClassKeyOfStyles } from "@mui/styles/withStyles";
-import { ClassNameMap } from "@mui/styles/withStyles/withStyles";
+import { Button, ButtonClassKey, buttonGroupClasses, ButtonProps, ComponentsOverrides } from "@mui/material";
+import { css, styled, Theme, useThemeProps } from "@mui/material/styles";
 import * as React from "react";
 import { FormattedMessage } from "react-intl";
 
 import { messages } from "../../../messages";
 import { useSplitButtonContext } from "../split/useSplitButtonContext";
-import { SaveButtonClassKey, styles } from "./SaveButton.styles";
+
+export type SaveButtonClassKey = "root" | "saving" | "error" | "success" | "conflict" | ButtonClassKey;
```

Comment:
`root` is already defined in `ButtonClassKey`

```suggestion
export type SaveButtonClassKey = "saving" | "error" | "success" | "conflict" | ButtonClassKey;
```